### ISSUE TITLE
Denpendencies version missed

### ISSUE THREAD

**Issue by H-xie** [ZwickyTransientFacility/ztf_sim#4]
Hi, I am reading your fantastic paper *The Zwicky Transient Facility: Surveys and Scheduler* and trying to replicate the experiment. 

The `sky model` in format `pkl` is `XGBoost` 's production, and it forces a version assertion. Could you provide the specific version of XGBoost and other dependencies (like numpy, sklearn, and so on) for replication? With the development of all those packages, their api and function turns different.

Thank you.😆

---

PS: This is the error message I met.

```
 If you are loading a serialized model (like pickle in Python) generated by older
  XGBoost, please export the model by calling `Booster.save_model` from that version
  first, then load it back in current version.  There's a simple script for helping
  the process. See:

    https://xgboost.readthedocs.io/en/latest/tutorials/saving_model.html

  for reference to the script, and more details about differences between saving model and
  serializing.
```
